lib/diagnostic.js

```diff
@@ -5,17 +5,22 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (/* previous, current */) {
+const min = function (previous, current) {
+    let array = [];
+      let minVal = Math.min(array[i]);
+
+    }; return(min);
```

Comment:
return statements must be INSIDE the function's {}
They will cause an error if they are outside.